src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -351,7 +351,17 @@ protected override void ProcessRecord()
                     HttpResponseMessage response = GetResponse(client, request);
                     response.EnsureSuccessStatusCode();
 
-                    string contentType = ContentHelper.GetContentType(response);
+                    //added try/catch due to possible lack of content type in the response header.
+                    //seems the only reason we are getting it here is for a log message?
```

Comment:
Current comments are pretty vague and suggest that author of the code is not completely sure about what the code does. It's totally ok for a codereview discussion, maybe even test code, but it should be avoided in the product code.

Think about the comments from the next-person-who-is-going-to-edit-it perspective.
The purpose of the comments is to **clarify** some aspects of the code, not to admit that code may be written incorrectly.